### PR TITLE
Responsive web

### PR DESCRIPTION
**Pull request by Wawrzyni3c**

### CODE REVIEW

File: cosqq.html [hunk 1]
```diff
@@ -1,166 +1,163 @@
...
+    <main>
+      <section id="hero">
+        <div class="welcome_text">
+          <h1>Lorem ipsum dolor</h1>
+          <h2>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit.Lorem ipsum
+            dolor sit amet consectetur adipisicing elit.Lorem ipsum dolor sit
+            amet consectetur adipisicing elit.
+          </h2>
+          <input class="subbmit_explore" type="submit" value="Explore" />
```
Comment (EnDriash) [inline cosqq.html]: [REQ]: Should be an button. You dont have any form here!

File: scss/sections/_contact.scss [hunk 2]
```diff
@@ -1,69 +1,190 @@
...
+          &::placeholder {
+            position: absolute;
+            top: 1rem;
+            font-weight: 600;
+            color: black;
+          }
+        }
+      }
+
+      .name_email {
```
Comment (EnDriash) [inline scss/sections/_contact.scss]: name email? bad naming !

File: scss/sections/_header.scss [hunk 3]
```diff
@@ -1,38 +1,91 @@
...
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+@media (max-width: 767px) {
+  body header .header_container {
```
Comment (EnDriash) [inline scss/sections/_header.scss:51]: why you need body here?

File: scss/sections/_informations.scss [hunk 4]
```diff
@@ -1,51 +1,102 @@
...
-		}
-	}
+  display: flex;
+  width: 100%;
+
+  .informations_container {
+    display: flex;
+    width: 100%;
+    max-width: 1400px;
+    margin: 0 auto 40px;
```
Comment (EnDriash) [inline scss/sections/_informations.scss]: margin should not have fixed dimensions!

File: scss/sections/_informations.scss [hunk 5]
```diff
@@ -1,51 +1,102 @@
...
+        width: 50%;
+        padding: 2% 5% 5% 0%;
+        color: black;
+        font-weight: normal;
+        text-align: left;
+
+        h1 {
+          color: black;
+          padding: 2% 0%;
+          margin-bottom: 15px;
```
Comment (EnDriash) [inline scss/sections/_informations.scss]: but for fonts and elements which contains text its ok to have fixed dimensions in padding/margin if its necessary.
it's necessary here to have margin here instead default margin?

File: scss/sections/_offer.scss [hunk 6]
```diff
@@ -1,50 +1,104 @@
...
+  margin: 10% 0;
+  flex-direction: column;
+  text-align: center;
+
+  h1 {
+    font-size: 50px;
+  }
+
+  h2 {
+    font-size: 26px;
```
Comment (EnDriash) [inline scss/sections/_offer.scss]: you sholud use rem or em instead px!!!
also for some of generic elements like this you should move this declarations for whole page to general.scss

File: scss/general.scss [hunk 7]
```diff
@@ -4,5 +4,5 @@
 @import "./sections/_hero.scss";
```
Comment (EnDriash) [inline scss/general.scss:5]: move this imports to index.scss
general file should contains only general styles for common elements.

File: scss/sections/_hero.scss [hunk 8]
```diff
@@ -1,54 +1,170 @@
...
-	border-top: 300px solid #005ad5;
-	border-right: 1802px solid transparent;
+@media (max-width: 767px) {
+  #hero {
+    display: flex;
+    width: 100%;
+    padding-top: 20%;
+    margin-top: -1%;
+    flex-direction: column;
+    // justify-content: center;
```
Comment (EnDriash) [inline scss/sections/_hero.scss]: [REQ]: remove comments

**Review by EnDriash — CHANGES_REQUESTED** [review]
Almost done ;)
You need some changes but really good job!

File: scss/sections/_header.scss (hunk 3 above)
Comment (Wawrzyni3c) [reply scss/sections/_header.scss:51]: because when i delate body my nav brakes down

**Pull request closed without merging**